jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -16079,96 +16018,25 @@ $as_echo "no (default for $OPENJ9_PLATFORM_CODE)" >&6; }
 
 
 
-  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for jitserver" >&5
-$as_echo_n "checking for jitserver... " >&6; }
   # Check whether --enable-jitserver was given.
 if test "${enable_jitserver+set}" = set; then :
   enableval=$enable_jitserver;
 fi
 
-  OPENJ9_ENABLE_JITSERVER=false
 
+  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for jitserver" >&5
+$as_echo_n "checking for jitserver... " >&6; }
+  OPENJ9_ENABLE_JITSERVER=false
   if test "x$enable_jitserver" = xyes ; then
-    { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes (explicitly enabled)" >&5
+    if test "x$OPENJDK_TARGET_OS" = xlinux ; then
+      { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes (explicitly enabled)" >&5
```

Comment:
similarly here, JITServer is supported on both plinux and xlinux.

Reply:
See above (this is a derived file).